indigo/tests/test_refs.py

```diff
@@ -33,6 +33,8 @@ def test_find_simple(self):
             <content>
               <p>Something to do with GN no 102 of 2012.</p>
               <p>And another thing about SI 4a of 1998.</p>
+              <p>don't match lowercase gn no 102 of 2012.</p>
+              <p>don't match where it's at the end of somethign no 102 of 2012.</p>
```

Comment:
shouldn't GN be uppercased here to test that it doesn't match mid-word? (minor)

Reply:
Good point, additional test added.